src/app/Processor/ProcessorEdit/ProcessorEdit.test.tsx

```diff
@@ -11,7 +11,12 @@ const setupProcessorEdit = (): {
   const onSave = jest.fn();
   const onCancel = jest.fn();
   const comp = customRender(
-    <ProcessorEdit onSave={onSave} onCancel={onCancel} isLoading={false} />
+    <ProcessorEdit
+      onSave={onSave}
+      onCancel={onCancel}
+      isLoading={false}
+      saveButtonLabel="Create"
+    />
```

Comment:
@vpellegrino can you add tests here?

Reply:
@sjamboro to test what? This PR is about plugging API, that will be covered by e2e tests

Reply:
@vpellegrino you have added new `props` (processorTypeSection - https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/6cae153cea074c1510dbf58e73cf7899f86f5f7e/src/app/Processor/ProcessorEdit/ProcessorEdit.tsx#L188-L190). So I expect that it could affect the view of the form.

Reply:
@sjamboro added https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/54/commits/8b673a16c596dbd0aadffe528cbce72e728d3267